Source:   Sterling X. Winter <sterling.winter@gmail.com>
Upstream: http://code.google.com/p/gtk-qt-engine/issues/detail?id=43
Reason:   Compilation fails on undefined function 'abs'

diff -ruN gtk-qt-engine.orig/src/qt_theme_draw.c gtk-qt-engine/src/qt_theme_draw.c
--- gtk-qt-engine.orig/src/qt_theme_draw.c	2008-03-10 20:30:19.000000000 -0500
+++ gtk-qt-engine/src/qt_theme_draw.c	2009-04-24 09:52:53.000000000 -0500
@@ -43,6 +43,9 @@
 
 #define GTK_QT_STANDARD_ARGS window, style, stateType, x, y, w, h, GTK_IS_WIDGET(widget) && gtk_widget_is_focus(widget)
 
+#ifndef abs
+#define abs(x) ((x)>=0?(x):(x*-1))
+#endif
 #ifndef max
 #define max(x,y) ((x)>=(y)?(x):(y))
 #endif
